MLIR.cmake

```diff
@@ -142,6 +144,21 @@ function(add_onnx_mlir_library name)
     target_link_libraries(${name} ${ARG_LINK_LIBS})
   endif()
 
+  if (ARG_LINK_COMPONENTS)
+    set(LinkageMode)
+    if (ARG_LINK_COMPONENTS MATCHES "^(PUBLIC|PRIVATE|INTERFACE)")
```

Comment:
I would add a comment in the description of the parameter that only one linkage mode can be specified.

Reply:
Done.